src/Microsoft.DotNet.CodeAnalysis/PackageFiles/codeAnalysis.targets

```diff
@@ -17,9 +17,7 @@
        2. For all other windows configuration we use OneCoreAPIs.txt file.
     -->
 
-    <ValidPinvokeMappings Condition="'$(TargetGroup)'==''">$(MSBuildThisFileDirectory)PinvokeAnalyzer_OneCoreApis.txt</ValidPinvokeMappings>
-    <ValidPinvokeMappings Condition="'$(TargetGroup)'=='netstandard13aot'">$(MSBuildThisFileDirectory)PinvokeAnalyzer_OneCoreApis.txt</ValidPinvokeMappings>
-    <ValidPinvokeMappings Condition="'$(TargetGroup)'=='netstandard15aot'">$(MSBuildThisFileDirectory)PinvokeAnalyzer_OneCoreApis.txt</ValidPinvokeMappings>
+    <ValidPinvokeMappings>$(MSBuildThisFileDirectory)PinvokeAnalyzer_OneCoreApis.txt</ValidPinvokeMappings>
```

Comment:
What it does right now is launch the P/Invoke scanner, but with no input source it rejects everything as prohibited.

That said, I don't actually know how to verify this for real... just edit the extracted copy in Tools? Or is there a better approach? (I hope "better approach" :smile:)